academia/modules.txt

```diff
@@ -1 +1,2 @@
-Academia
\ No newline at end of file
+Academia
+Council
```

Comment:
Rename the module into "Councils"

Reply:
Thank you for your feedback. I will rename the module from 'Council' to 'Councils' as requested. I'll make the necessary changes and update the pull request accordingly. Please re-review it once the updates are complete. Thank you for your time.

Reply:
changes  applied